jak2/opengoal-jak2-autosplitter.asl

```diff
@@ -151,7 +150,7 @@ startup {
   AddToSettings(vars.allMissions, "preset_allMissions");
   vars.optionLists.Add(vars.allMissions);
   vars.hundredPercent = new List<Dictionary<String, dynamic>>();
-  AddOption(vars.hundredPercent, "hundredPercent_res-fortress-escape", 418, typeof(byte), 1, false, "res-fortress-escape", true);
+  AddOption(vars.hundredPercent, "hundredPercent_res-fortress-escape", 418, typeof(byte), 1, false, "res-fortress-escape", false);

```

Comment:
@xTVaser were these left as debug for any important reason? I just re-ran the python script and it flipped them back to debug=false